lib/nifty_services/support/string.rb

```diff
@@ -0,0 +1,14 @@
+unless String.method_defined?(:underscore)
+  class String
+   def underscore
+      return self unless self =~ /[A-Z-]|::/
+      word = self.to_s.gsub('::'.freeze, '/'.freeze)
+      word.gsub!(/(?:(?<=([A-Za-z\d]))|\b)(#{inflections.acronym_regex})(?=\b|[^a-z])/) { "#{$1 && '_'.freeze }#{$2.downcase}" }
```

Comment:
@lucasgomide Right, thanks, I get it! But keeping this we still depend on ActiveSupport, right?